scripts/git/common.ps1

```diff
@@ -55,9 +55,9 @@ function Get-StagedFiles {
 
 # Formats the given file using ClangFormat
 function ClangFormatFile([string]$repoRoot, [string]$filename, [bool]$isDryRun){
-    $clangCommand = "$repoRoot/tools/scripts/clang-format.ps1"
+    $clangCommand = "$repoRoot/scripts/git/clang-format.ps1"
```

Comment:
what's the plan for getting this into develop then? are we cherry-picking it, or will develop not have it until packaging merges?